.github/ISSUE_TEMPLATE/release.md

```diff
@@ -40,17 +40,11 @@ production readiness.
 - [ ] Choose the commit for the release. Unless there are open release blockers above,
   this should just be the head commit of the main branch. Checkout that commit.
 
-- [ ] Review all PRs in the release that were not reviewed by another engineer
-  prior to being merged into main.
-
-- [ ] Choose the name for the release candidate following the format
-  `v<VERSION>-rc<N>`, where _N_ starts at 1. For example, the first RC for
-  v0.2.3 would be called v0.2.3-rc1. Use that to run the `bin/mkrelease`
-  script:
+- [ ] Create a new release candidate, specifying what kind of release this
+  should be (see `--help` for all the release types):
 
   ```shell
-  tag=v<VERSION>-rc<N>
-  bin/mkrelease -b rel-$tag $tag
+  bin/mkrelease new-rc patch
```

Comment:
I've found that setting the version in this and all the other commands gets kind of annoying. 

There is always only one correct choice for every command, so while the error checking doesn't allow you to specify the wrong thing it is also easy to mechanically determine the right thing, which IMO means it should be mechanically determined. This API also goes through all the other commands here, the cumulative effect might be small but in testing it's much more pleasant.

If it's confusing I can add some feature that allows you to specify the tag instead of the bump kind.

Reply:
Are we intentionally committing to SemVer language by calling this "patch" ? It's not really a patch release in the normal English meaning of that term.

(Opened up a thread in #release on Slack to discuss this)

Reply:
no, this is purely an internal-facing feature and "patch" seemed like a fine word since we don't have another word to describe this part of the version number.